lib/platforms/browser.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var React = require('react');
+
+var CONTAINER_ID = 'fluxapp-container';
+
+function renderComponent(routeStore, Component, serverSideRendered, routeId) {
+  if (routeStore.state.current.id === routeId) {
+    React.render(React.createElement(Component, {
+      serverSideRendered : true
+    }), document.getElementById(CONTAINER_ID));
+  }
+}
+
+module.exports = function initBrowser(fluxApp) {
+  var router = require('fluxapp-router');
```

Comment:
router can only run in the browser and will throw exceptions on node, we don't want it included on the server side